### ISSUE TITLE
Document that boolean flags take a value

### ISSUE THREAD

**Issue by izaaklauer** [hashicorp/waypoint#3049]
Currently, it is undocumented that boolean flags can take a value. Example, the `waypoint server run -url-enabled` flag:

https://www.waypointproject.io/commands/server-run#url-enabled

The only way to turn off the url service is to set `-url-enabled=false` , but we don't document that the flag even takes a bool.

This doesn't matter for most of our flags, as generally we set the default value of boolean flags to false. e.g. `-disable-ui`. If you want the UI, you ignore the flag, and if you don't, you set `-disable-ui` with no args - the shorthand is all you need.

**Describe the solution you'd like**
I think we should update our auto doc generation for bool flags to always say `-flag=<bool>`. This means we won't be documenting the shorthand version anymore, but at least this way we're documenting the full functionality.

**Describe alternatives you've considered**

- We could change all our flags so the default is always false. I.e. deprecate `-url-enabled` in favor of `-url-disabled`.
- We could be extra smart in the doc generator and only add the bool arg when the default value is `true`.

**Comment by briancain** [comment]
Agreed, having truthy flags seems like a CLI anti-pattern. Probably don't have to change existing flag names yet, but this seems like a good UX design style guide entry item to follow in the future going forward!

**Comment by mitchellh** [comment]
My argument would be that all boolean flags should always have the same truthyness regardless of default value, because it makes it consistent without having to know semantically how Waypoint will act. Also, if you change the default in the future, it gets weird cause the name pattern of your flags is tied to your behavior.

I'm supportive of making that consistent.
